personal_assistant/pa_note/templates/pa_note/note_list.html

```diff
@@ -0,0 +1,19 @@
+{% extends "pa_core/home.html" %}
+
+{% block content %}
+    <h1>List of notes</h1>
+    <a href="{% url 'pa_note:add_note' %}">Add note</a>
+    <ul>
+        {% for note in notes %}
+            <li>
+                {{ note.name }} - {{ note.description }} -
+                {% if note.done %} Done {% else %} Not done {% endif %}
+                <a href="{% url 'pa_note:edit_note' note.id %}">Edit</a> <!-- Ссылка на редактирование -->
+                <form action="{% url 'pa_note:delete_note' note.id %}" method="post" style="display:inline;">
+                    {% csrf_token %}
+                    <button type="submit">Delete</button>
+                </form>
```

Comment:
Пропоную зробити значно простіше проте це хороша ідея (запитувати про підтвердження на видалення) тож раджу це зробити не як окрему сторінку, а більш сучасно - [модальним вікном](https://getbootstrap.com/docs/5.3/components/modal/#live-demo):
```suggestion
                <a href="{% url 'pa_note:delete_note' note.id %}">Delete</a>
```